auslib/db.py

```diff
@@ -952,19 +952,13 @@ def getReleaseInfo(self, product=None, limit=None,
         if not nameOnly:
             j = join(dbo.releases.t, dbo.rules.t, ((dbo.releases.name == dbo.rules.mapping) | (dbo.releases.name == dbo.rules.whitelist)))
             ref_list = select([dbo.releases.name, dbo.rules.rule_id]).select_from(j).execute().fetchall()
-            ref_dict = {}
-
-            for ref in ref_list:
-                rel_name, rule_id = ref
-                try:
-                    ref_dict[rel_name].append(rule_id)
-                except KeyError:
-                    ref_dict[rel_name] = [rule_id]
 
             for row in rows:
-                try:
-                    row['rule_ids'] = ref_dict[row['name']]
-                except KeyError:
+                refs = [ref for ref in ref_list if ref[0] == row['name']]
```

Comment:
I don't think "ref for ref" is safe to do...those are two different variables that you're using the same name for. At the very least, it's confusing. You could use something like "r for ref in ref_list" instead.